core/field_number.js

```diff
@@ -32,10 +32,10 @@ goog.require('Blockly.utils.object');
  * Class for an editable number field.
  * @param {string|number=} opt_value The initial value of the field. Should cast
  *    to a number. Defaults to 0.
- * @param {(string|number)=} opt_min Minimum value.
- * @param {(string|number)=} opt_max Maximum value.
- * @param {(string|number)=} opt_precision Precision for value.
- * @param {Function=} opt_validator A function that is called to validate
+ * @param {?(string|number)=} opt_min Minimum value.
+ * @param {?(string|number)=} opt_max Maximum value.
+ * @param {?(string|number)=} opt_precision Precision for value.
+ * @param {?Function=} opt_validator A function that is called to validate
```

Comment:
I'd leave this one out unless the super class is ready to handle null in it's types. FieldTextInput doesn't say that it handles a null validator (currently).

Reply:
It has to handle null if the dev is using the config object, as it's not reasonable to ask devs to pass undefined instead of null to get to config.